src/sinks/honeycomb.rs

```diff
@@ -251,4 +302,106 @@ mod test {
         let event = Event::Log(LogEvent::from("simple message"));
         run_and_assert_sink_compliance(sink, stream::once(ready(event)), &HTTP_SINK_TAGS).await;
     }
+
+    #[test]
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Thanks for adding unit tests!